plsync/mlabconfig.py

```diff
@@ -45,13 +64,152 @@ def parse_flags():
         default='slice_list',
         help=('The variable name of experiments within the experiments_config '
               'data.'))
-    # TODO: Support multiple formats, e.g. 'hostips', 'zone', 'sitestats'.
     parser.add_option(
         '', '--format', metavar='format', dest='format', default='hostips',
         help='Format of output.')
+    parser.add_option(
+        '', '--zoneheader', metavar='zoneheader.in', dest='zoneheader',
+        default=ZONE_HEADER_TEMPLATE,
+        help='The full path to zone header file.')
     return parser.parse_args()
 
 
+def comment(output, note):
+    output.write('\n; %s\n' % note)
+
+
+def write_a_record(output, hostname, ipv4):
+    output.write(format_a_record(hostname, ipv4))
```

Comment:
It's a good question.

Originally it was just one function, and when I wrote the tests, it was helpful to have the format function separate from the write function.

So, my thought is that separating the functions make testing easier.